ERCS/erc-0000.md

```diff
@@ -0,0 +1,103 @@
+---
+eip: erc-0000.md
```

Comment:
```suggestion
eip: 7744
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.